lib/matplotlib/axes/_axes.py

```diff
@@ -2822,7 +2822,10 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         plot_line = (fmt.lower() != 'none')
         label = kwargs.pop("label", None)
 
-        fmt_style_kwargs = {k: v for k, v in
+        if fmt=='':
```

Comment:
the PEP8 style checker wants this to be: `if fmt == ''` (spaces around the operator)

Reply:
Test is simple

~~~python
import matplotlib.pyplot as plt
plt.rc("axes", prop_cycle = 'cycler(linestyle=["--",":"],mfc=["k","w"],marker=["s","s"])')
plt.errorbar(x=[1,3,4],y=[3,2,3],yerr=0.5)
plt.errorbar(x=[1,3,4],y=[1,0,1],yerr=0.5)
plt.show()
~~~
I am exceptionally uncomfortable with git, github and compiling matplotlib, at least on windows. I just fixed the file axes/_axes.py in my winpython installation with matplotlib 2.1.0 and it worked. I hope it will be easier to compile matplotlib  on Linux. If it works I will be able to contribute in  more appropriate and civilized way.

Reply:
Understood. Do you mind if I add a commit with a test and push to your branch later today?

Reply:
I don't mind anything. I thought I can save somebody's time because I've already found the solution. I do think it is very handy to be able to contribute properly instead of working  around the issues, just don't have much time right now to invest in troubleshooting my development setup. It compiles and installs, but then matplotlib complains it can't find some dll. So if you have it up and running, I hope you can test my changes against the bleeding edge before adding them. Thanks for being patient with me.